wagtail/contrib/forms/views.py

```diff
@@ -64,13 +66,50 @@ class FormPagesListView(SafePaginateListView):
     def get_queryset(self):
         """ Return the queryset of form pages for this view """
         queryset = get_forms_for_user(self.request.user)
+        if self.locale:
+            queryset = queryset.filter(locale=self.locale)
         ordering = self.get_ordering()
         if ordering:
             if isinstance(ordering, str):
                 ordering = (ordering,)
             queryset = queryset.order_by(*ordering)
         return queryset
 
+    def get(self, request, *args, **kwargs):
+        self.locale = None
+        enable_locale_filter = getattr(settings, 'WAGTAIL_I18N_ENABLED', False)
+        if enable_locale_filter:
+            if request.GET.get('locale'):
+                self.locale = get_object_or_404(Locale, language_code=request.GET['locale'])
+            else:
+                self.locale = Locale.get_default()
+
+        return super().get(request, *args, **kwargs)
+
+    def get_context_data(self, *, object_list=None, **kwargs):
+        local_context = {
```

Comment:
nitpick: missing an `e`

Reply:
updated and rebased on `main` while at it

Reply:
Ohh, I see, but thanks for changing it anyway!